src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1AssessmentEmailService.kt

```diff
@@ -41,4 +49,38 @@ class Cas1AssessmentEmailService(
       )
     }
   }
+
+  private fun deadlineCopy(deadline: OffsetDateTime?, isEmergency: Boolean): String {
+    if (deadline == null) {
+      return DEFAULT_DEADLINE_COPY
+    }
+
+    if (isEmergency) {
+      val deadlineDate = deadline.toLocalDate()
+      return if (deadlineDate.isEqual(LocalDate.now())) {
+        SAME_DAY_EMERGENCY_DEADLINE_COPY
+      } else {
+        NEXT_WORKING_DAY_EMERGENCY_DEADLINE_COPY.format(deadline.toUiFormattedTime(), deadlineDate.toUiFormat())
+      }
+    }
+
+    return STANDARD_DEADLINE_COPY.format(
+      workingDayCountService.getWorkingDaysCount(LocalDate.now(), deadline.toLocalDate()).toString(),
```

Comment:
above you're using toUiFormat for a date, but here using .toString(). Not sure if that's deliberate?

Reply:
Yeah, `getWorkingDaysCount` returns an integer, so we're casting to a string to be used in the template.